ui/apps/platform/src/Containers/VulnMgmt/List/Cves/ListCVEs.utils.js

```diff
@@ -55,7 +58,7 @@ export function getFilteredCVEColumns(columns, workflowState) {
                 return shouldKeepEntitiesColumn;
             }
             case 'severity': {
-                return shouldKeepDiscoveredAtImageColumn;
+                return shouldKeepSeverity || shouldKeepDiscoveredAtImageColumn;
```

Comment:
why are you checking for `shouldKeepDiscoveredAtImageColumn` to determine the visibility of severity?

Reply:
It seems like we use that to determine whether to show the `severity` column. If you git blame you can track it down to this PR https://github.com/stackrox/rox/pull/8080. It seems like he added it according to that PR. All I know is that we want to show the severity for the Image CVEs and Node CVEs tables so I'm adding that conditional logic to display it for those. I don't want to mess around too much with what's here right now since we will eventually make changes to VM soon. If you want to dig into why it was added, maybe you can ask Gorman for clarification.